downstream/modules/automation-hub/proc-using-content-signing-services-in-pah.adoc

```diff
@@ -0,0 +1,35 @@
+[id="proc-using-content-signing-services-in-pah_{context}"]
+
+= Using content signing services in private automation hub
+
+After you have configured content signing on your {PrivateHubName}, you can manually sign a new collection or replace an existing signature with a new one so that users who want to download a specific collection have the assurance that the collection is intended for them and has not been modified after certification.
+
+Content signing on {PrivateHubName} provides solutions for the following scenarios:
+
+* Your system does not have automatic signing configured and you must use a manual signing process to sign collections.
+* The current signatures on the automatically configured collections are corrupted and must be replaced with new signatures.
+* Additional signatures are required for previously signed content.
+* You want to rotate signatures on your collections.
+
+.Procedure
+
+. In the `automationhub-terminal`, run the following command to publish your collections:
```

Comment:
This is unnecessary

Reply:
So the entire first step along with the code sample should be removed, correct?

Reply:
yeah `automationhub-terminal` is something I named to the terminal in my lab, it has nothing to do with the feature, and more of a self-paced lab construct, where I say this is where you run commands.
for an actual customer that is just a terminal or backend of private automation hub host.